source/components/model/edges/edgeDefinition.tsx

```diff
@@ -0,0 +1,266 @@
+import { getStraightPath, BaseEdge, EdgeLabelRenderer, MarkerType, getMarkerEnd } from "reactflow";
+
+
+export function edgeTagMapper(edgeTag:string):string{
+  switch(edgeTag){
+    case 'reads': return 'readsEdge'
+    case 'defined-by': return 'definedByEdge'
+    case 'same-read-read': return 'sameReadReadEdge'
+    case 'same-def-def': return 'sameDefDefEdge'
+    case 'calls': return 'callsEdge'
+    case 'returns': return 'returnsEdge'
+    case 'defines-on-call': return 'definesOnCallEdge'
+    case 'defined-by-on-call': return 'definedByOnCallEdge'
+    case 'argument': return 'argumentEdge'
+    case 'side-effect-on-call': return 'sideEffectOnCallEdge'
+    case 'relates': return 'relatesEdge'
+  }
+  return ''
+}
```

Comment:
Please use a record like this (you should be able to just apply the suggestion). You can use a map as well if you want:

```suggestion
const edgeTagMap = {
  'reads':               'readsEdge',
  'defined-by':          'definedByEdge',
  'same-read-read':      'sameReadReadEdge',
  'same-def-def':        'sameDefDefEdge',
  'calls':               'callsEdge',
  'returns':             'returnsEdge',
  'defines-on-call':     'definesOnCallEdge',
  'defined-by-on-call':  'definedByOnCallEdge',
  'argument':            'argumentEdge',
  'side-effect-on-call': 'sideEffectOnCallEdge',
  'relates':             'relatesEdge'
};

function edgeTagMapper(edgeTag: string): string {
  return edgeTagMap[edgeTag] ?? '';
}
```